polling_stations/settings/static_files.py

```diff
@@ -6,22 +6,43 @@
     extra_js=[],
 )
 
-PIPELINE["STYLESHEETS"]["map"] = {
-    "source_filenames": [
-        "leaflet/dist/leaflet.css",
-        "custom_css/map.css",
-        "leaflet-extra-markers/dist/css/leaflet.extra-markers.min.css",
-    ],
-    "output_filename": "css/map.css",
-}
+PIPELINE["STYLESHEETS"].update(
+    {
+        "map": {
+            "source_filenames": [
+                "leaflet/dist/leaflet.css",
+                "custom_css/map.css",
+                "leaflet-extra-markers/dist/css/leaflet.extra-markers.min.css",
+            ],
+            "output_filename": "css/map.css",
+        },
+        "dashboard": {
+            "source_filenames": [
+                "leaflet.markercluster/dist/MarkerCluster.css",
+                "leaflet.markercluster/dist/MarkerCluster.Default.css",
+            ],
+            "output_filename": "css/map.css",
```

Comment:
This bundle should be called `dashboard.css`

Reply:
Ah yes. Thanks!